src/components/Button/index.stories.tsx

```diff
@@ -60,7 +62,7 @@ Inverted.parameters = {
   },
 };
 
-export const isActive = () => (
+export const isActive = (): JSX.Element => (
```

Comment:
I've been typing the component as opposed to the return type:
```suggestion
export const isActive: React.FC = (): JSX.Element => (
```
`React.FC` is an alias for `React.FunctionComponent`. This could be more flexible because you can add props later if you need to, i.e. `React.FC<ComponentPropInterface>`.
The return type is also _slightly_ different. It returns a `ReactElement | null`.  `ReactElement`'s definition looks like this:
```typescript
interface ReactElement<P = any, T extends string | JSXElementConstructor<any> = string | JSXElementConstructor<any>> {
    type: T;
    props: P;
    key: Key | null;
}
```

I only bring all this up because it might be good to be consistent about this sort of thing.

Reply:
Any thoughts on what this resource says? https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components/ (it has reasoning why it recommends against React.FC down below)

Reply:
I'm on board with the reasons brought up in that article and in secondary links. My two main takeaways:
- it's more explicit what you want the return to me. Some people might see it as a chore, but it'll hopefully help them understand better.
- The `React.FunctionComponent` by default allows your component to accept `children` even if you don't provide them in the props. That's just misleading.

I think the main reason I've always used `React.FC` is because when I first started using TS + React I found it hard to figure out which return type to use: `React.Element`, `React.ElementType`, `React.ReactNode`. They all sound vaguely appropriate if you're just searching through autocomplete for what to use, but I'd never guess `JSX.Element`, so when I found that `React.FC` just took care of it for me, I started using that.